Gems/ScriptCanvas/Code/Include/ScriptCanvas/Translation/GraphToLua.cpp

```diff
@@ -962,7 +1045,10 @@ namespace ScriptCanvas
                 for (auto& out : nodeAndParse->m_latents)
                 {
                     m_dotLua.WriteNewLine();
-                    TranslateNodeableOut(nodeAndParse->m_nodeable, out.second);
+                    TranslateExecutionOut
```

Comment:
This was the location of the original bug. At a certain point, when the distinction between a FunctionDefinitionNode and a FunctionCallNode were split from Nodeling, one distinction was lost, and it was here. So then the wrong implementation of calling a user out was used. The interpreter used the one intended for C++ nodeables exposed to the SC runtime, rather than the one intended for classes written as the result of SC graphs themselves.